custom-pages/case-autogenerated-overview/.jshintrc

```diff
@@ -0,0 +1,40 @@
+{
+  "node": true,
+  "browser": true,
+  "esnext": true,
+  "bitwise": true,
+  "camelcase": true,
+  "curly": true,
+  "eqeqeq": true,
+  "immed": true,
+  "indent": 2,
+  "latedef": true,
+  "newcap": true,
+  "noarg": true,
+  "quotmark": "single",
+  "regexp": true,
+  "undef": true,
+  "unused": true,
+  "strict": true,
+  "trailing": true,
+  "smarttabs": true,
+  "globals": {
+    "$": false,
+    "after": false,
+    "afterEach": false,
+    "angular": false,
+    "by": false,
+    "before": false,
+    "beforeEach": false,
+    "browser": false,
+    "describe": false,
+    "expect": false,
+    "inject": false,
+    "it": false,
+    "jasmine": false,
+    "require": false,
+    "spyOn": false,
+    "protractor":false,
+    "element":false
+  }
+}
```

Comment:
we have a jshintrc for every module? not possible to reference one in a single place?